src/main.rs

```diff
@@ -37,6 +47,29 @@ struct Params {
     bind: String,
 }
 
+/// Load datasets from the configuration
+/// if a config file has been given, we get the dataset from here,
+/// else we read the gtfs/url cli parameter to create a 'default' dataset with them
+fn get_datasets(params: &Params) -> Result<Datasets, failure::Error> {
+    if let Some(config) = &params.config_file {
+        let yaml = if config.starts_with("http") {
+            serde_yaml::from_reader(reqwest::get(config)?)
```

Comment:
wouldn't it be better to remove this `?` (and the one for `file::open`) ?
this way we could have an error with a bit more context (`"impossible to read config file because: ..."`)